web/typescript/page/edit.ts

```diff
@@ -0,0 +1,9 @@
+export function checkMaxLength() {
+    [...document.querySelectorAll('textarea[maxlength]')].forEach((element) => {
```

Comment:
Give it a correct type:
```suggestion
    [...document.querySelectorAll('textarea[maxlength]')].forEach((element: HTMLTextAreaElement) => {
```

Reply:
Do we still need to query for 'textarea' after that?

Reply:
Not sure where we query for "textarea" afterwards. This is just a type annotation to let TypeScript know what we're working with. It inferred `element` as `HTMLElement`, so that wasn't ideal. I changed it to a union type later since both `input` and `textarea` have `maxlength` attributes.